### DISCUSSION TITLE
[5.x] Make breadcrumbs sticky or visible when scrolling to top

### DISCUSSION THREAD

**Discussion by white-lukas** [craftcms/cms#15320] [Ideas]
In Craft 3 and 4 the dropdown to switch sites was part of the searchbar and was easily accessible.
Now in Craft 5 when in on an overview or detail page, when you want to switch sites, you have to scroll back to the top, to view the dropdown. it would be easier if the breadcrumb bar is also sticky, or re-appears when you start scrolling back

**Comment by ammannbe** [comment]
I also realized a similar behaviour during editing/creating entries.
When scrolling down and edit content, you have to scroll back to the top to Discard the edited changes
